### PR TITLE
Add com.voxdsp.Borg

### PR DESCRIPTION
**Pull request by BeverlyCode**
### Please confirm your submission meets all the criteria

- [ ] Please describe your application briefly.
- [ ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ ] My pull request follows the instructions at [App Submission][submission].
- [ ] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by 8smcrlgq850irkn7pmvjzg1s** [comment]
Seems I cant automatically merge my own changes anymore, that's changed too I guess lol?

**Comment by bbhtt** [comment]
Please complete the checklist.

File: com.voxdsp.Borg.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+app-id: com.voxdsp.Borg
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```
Comment (bbhtt) [inline com.voxdsp.Borg.yaml:3]: 23.08 is the latest

File: com.voxdsp.Borg.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+app-id: com.voxdsp.Borg
```
Comment (bbhtt) [inline com.voxdsp.Borg.yaml:1]: This time, do you own the website? Will you be able to verify it if needed?

File: com.voxdsp.Borg.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: borg
+    buildsystem: simple
+    build-commands:
+      - cc main.c -Ofast -lSDL2 -lm -o borg
+      - install -Dm 0755 borg -t ${FLATPAK_DEST}/bin
+      - install -Dm 0644 flatpak/borg.desktop ${FLATPAK_DEST}/share/applications/borg.desktop
+      - install -Dm 0644 flatpak/borg.appdata.xml ${FLATPAK_DEST}/share/metainfo/borg.appdata.xml
```
Comment (bbhtt) [inline com.voxdsp.Borg.yaml:22]:
```suggestion
      - install -Dm 0644 flatpak/borg.appdata.xml ${FLATPAK_DEST}/share/metainfo/borg.metainfo.xml
```

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#path-and-filename

**Comment by hfiguiere** [comment]
Isn't this already on flathub?

**Comment by hfiguiere** [comment]
https://github.com/flathub/com.voxdsp.Borg

**Pull request closed without merging**